src/styles/convertkit.scss

```
@@ -2,6 +2,7 @@
 	border-radius: 5px;
 	padding: 20px;
 	margin: 1rem auto;
+	// TODO(undefined-css-vars): no similar token found for --cardOutlineStyle
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)